internal_file_registry_service/config.py

```diff
@@ -15,23 +15,18 @@
 
 """Config Parameter Modeling and Parsing"""
 
-from functools import lru_cache
-
 from ghga_service_chassis_lib.config import config_from_yaml
+from ghga_service_chassis_lib.postgresql import PostgresqlConfigBase
 from ghga_service_chassis_lib.pubsub import PubSubConfigBase
+from ghga_service_chassis_lib.s3 import S3ConfigBase
 
 
-@config_from_yaml(prefix="internal-file-registry-service")
-class Config(PubSubConfigBase):
+@config_from_yaml(prefix="internal_file_registry_service")
```

Comment:
shouldn't we keep the yaml file to be able to change standard parameters when needed?

Reply:
Will do this in a seperate PR

Reply:
What do you mean? This decorator is exactly doing that.

Reply:
I mean that the yaml file was deleted in this PR. Apparently you can't comment on deleting an empty file :-D

Reply:
Ah I see. It was replaced by the `.devcontainer/.dev_config.yaml` and is ingested via the INTERNAL_FILE_REGISTRY_SERVICE_CONFIG_YAML. These changes where made so that we don't have to be in the `.devcontainer` directory when running tests or anything else that runs the service's code.